src/plugins/Sharesite/Inserter.java

```diff
@@ -60,6 +83,18 @@ public void run() {
 		}
 	}
 
+	private synchronized Freesite getNextToInsert(Integer currentHour, LinkedList<Freesite> queuedInserts) {
+		for(Freesite c : queuedInserts) {
```

Comment:
it’s the *c*urrently checked Freesite in a tight loop (so the meaning of the variable should be clear). Is that overly surprising to java devs? If yes, what would you suggest?